src/ontology/edits/oeo-physical.omn

```diff
@@ -3066,8 +3066,11 @@ Class: OEO_00010024
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000118> "BEV",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/425
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/453",
-        dc:description "A battery electric vehicle (BEV) is a vehicle that stores energy in an on-board battery.",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/453
+
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/637
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/655",
+        dc:description "A battery electric vehicle (BEV) is an electric vehicle that stores energy in an on-board battery.",
```

Comment:
It seems like `dc:description` had been used for the definition here. Could you change that to `http://purl.obolibrary.org/obo/IAO_0000115` (our current definition-annotation)?